src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -321,4 +323,23 @@ public static void verifyColumnDisplayOrder(String column1Name, String tableName
         assertTrue(expectedResult, table, () -> table.verifyColumnDisplayOrder(column1Name, column2Name));
     }
 
+    /**
+     * Verify the element exists in row which have stored value in column name
+     * <p>
+     * <b>Gherkin Examples:</b>
+     * <ul>
+     * <li>I verify row in the Project Files Table with value selected for the Search Input contains the xpath //i[contains(concat(' ', @class, ' '), ' fa-toggle-on ')]</li>
+     * </ul>
+     * @param tableName String the name of the table element
+     * @param key String the key used to retrieve the value
+     * @param xpath String xpath value to element to click
+     */
+    @Then("^I verify row in the (.*?) with value (?:entered|selected|used) for the (.*?) contains the xpath (.*?)$")
+    public static void verifyStoredTextRowContainsXpath(String tableName, String key, String xpath) {
```

Comment:
this doesn't seem to follow the assertion / verification pattern we have in most of the other TableVerificationSteps. I'd like to see us create a table method that returns a boolean, and then keep that verification pattern going where we use `assertTrue(errorMessage, table, table -> table.verificationMethod)`  I'd also like to see the negative outcome covered, in the case that we want to verify a certain row does not contain an element matched by an xpath.